### ISSUE TITLE
[adev] Add Discord link icon in navigation menu

### ISSUE THREAD

**Issue by geromegrignon** [angular/angular#56097]
### Describe the problem that you experienced

The angular.dev navigation menu includes the list of social media except the Discord one:

<img width="191" alt="image" src="https://github.com/angular/angular/assets/32737308/b8ce5fbb-46a7-4763-8ce6-f3fd76359721">

### Enter the URL of the topic with the problem

https://angular.dev/

### Describe what you were looking for in the documentation

The Discord server link is only visible in the footer, not within the navigation menu

### Describe the actions that led you to experience the problem

Click on the '3 dots' icon button in the navigation menu.

### Describe what you want to experience that would fix the problem

Add a link icon for the Discord Server

### Add a screenshot if that helps illustrate the problem

<img width="191" alt="image" src="https://github.com/angular/angular/assets/32737308/2a190b66-4225-45d2-9d88-d5ebaec23127">

**Comment by vidhihattar** [comment]
I'd like to work on this. May I ask where I can get the appropriate discord svg?

**Comment by varaprasad6699** [comment]
@JeanMeche,
To work on this can we get some information like
1. Discord icon link 
2. Need Url that it should redirect after clicking on the Discord icon.
Thank you.

**Comment by geromegrignon** [comment]
The link: https://discord.gg/angular
The official Discord svg icon can be found here: https://discord.com/branding

Ensures to update its size as in the other icons: https://github.com/angular/angular/blob/main/adev/src/app/core/layout/navigation/navigation.component.html#L368

**Comment by vidhihattar** [comment]
I'd like to open a PR for this. For the commit message according to contribution guidelines there needs to be a type. This would fall under which category? Will this be a fix?

**Comment by geromegrignon** [comment]
The type is **docs** @vidhihattar

**Comment by JeanMeche** [comment]
@vidhihattar If you open a PR, we'll help you land it :)

**Comment by vidhihattar** [comment]
According to the readme.md in "adev" folder, we have to run "yarn docs" to start the local development server, which when I run the first time without any changes in the code, runs fine:
<img width="1031" alt="Screenshot 2024-05-28 at 3 39 40 PM" src="https://github.com/angular/angular/assets/84982665/3232bad2-1e92-4aee-8562-2064985a29e3">
But when I make the changes in the code and rerun the "yarn docs" command, it gets stuck here:
<img width="1066" alt="Screenshot 2024-05-28 at 3 40 43 PM" src="https://github.com/angular/angular/assets/84982665/9af5da08-e35e-4934-9257-8ea46deaf664">
Am I missing something? I have tried a few things suggested on the internet but nothing seems to work. I would appreciate any guidance you could give me.

**Comment by JeanMeche** [comment]
Yeah live update isn't working well, you need to restart after the changes.

**Comment by vidhihattar** [comment]
By restart you mean running "yarn docs" again ? because that is not working as well. As seen in the second screenshot I have attached in my previous message, it gets stuck.

**Comment by bencodezen** [comment]
This issue has been resolved and the Discord icon is now on the site!
